app/authorities/sufia/authorities/local/find_works.rb

```diff
@@ -0,0 +1,21 @@
+module Sufia::Authorities
+  class Local::FindWorks < Qa::Authorities::Base
+    attr_reader :q
+
+    def initialize(q)
```

Comment:
I assumed it was the required query variable for the QA.  @jcoyne had mentioned removing it. But in doing so, the code crashes.